src/plugins/custom_import_map/opensearch_dashboards.json

```diff
@@ -1,5 +1,5 @@
 {
-    "id": "customImportMap",
+    "id": "customImportMapDashboards",
```

Comment:
Since the id is changed can the folder of `src/plugins/custom_import_map` also changes to `src/plugins/customImportMapDashboards` to be consistent?

Reply:
That's not required - eg. https://github.com/opensearch-project/alerting-dashboards-plugin

Reply:
Added the followup issue here - https://github.com/opensearch-project/dashboards-maps/issues/28

Thanks @peterzhuamazon for the suggestion!

Reply:
Thanks @Shivamdhar we will leave this as an improvement later per our discussion offline.